rise-playlist.html

```diff
@@ -176,8 +168,8 @@
           duration = parseInt(duration, 10);
 
           if (!isNaN(duration)) {
-            setTimeout(function() {
-              self._playNext(index, "pause");
+            this.debounce("play-next", function() {
+              this._playNext(index, "pause");
```

Comment:
@stulees Here's a simple explanation that's better than what's in the docs - http://goo.gl/nOnmZU.

Since `play` is a public function and could be inadvertently called multiple times, I'm going to leave the `debounce` call in so I don't have multiple timers being started inadvertently. Thx.